src/main/java/seedu/address/model/transaction/SellTransaction.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.model.transaction;
+
+/**
+ * Store the sell transaction process.
+ */
+public class SellTransaction extends Transaction {
+    private static final double NEGATIVE = -1;
```

Comment:
Should this be ```POSITIVE = 1``` instead? Since we should be calculating the cashflow and selling should give positive cashflow.